algorithm_toolkit/utils/home_utils.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import json
+
+from flask import make_response
+
+from .data_utils import create_random_string
+from .file_utils import get_chain_def
+
+
+def chain_check(request, chain_name, path):
```

Comment:
maybe rename this "check_chain"

Reply:
will do

Reply:
Actually. I think `check_request_form` may be better? Thoughts?